### PR TITLE
BETA: Register ja3.is-a.dev

### PR DESCRIPTION
**Pull request by Ja3mamtora**
Added `ja_3.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by Ja3mamtora** [comment]
@WilliamDavidHarrison Please can you review this

File: domains/ja_3.json
Comment (DEV-DIBSTER) [inline domains/ja_3.json:1]: Invalid file name, sub domains cannot contain _.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by Ja3mamtora** [comment]
@DEV-DIBSTER I have removed for ja_3 and added for Ja3mamtora can you please review it

File: domains/ja3mamtora.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Ja3mamtora",
+        "email": "jatinumamtora@gmail.com"
+    },
+    "record": {
+        "CNAME": "ja3mamtora.github.io"
```
Comment (DEV-DIBSTER) [inline domains/ja3mamtora.json:7]: CNAME returns 404 record.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/ja3mamtora.json (hunk 1 above)
Comment (Ja3mamtora) [reply domains/ja3mamtora.json]: check it now

**Review by Ja3mamtora — COMMENTED** [review]
Check it now

File: domains/ja3mamtora.json [hunk 2]
```diff
@@ -4,6 +4,6 @@
         "email": "jatinumamtora@gmail.com"
     },
     "record": {
-        "CNAME": "ja3mamtora.github.io"
+        "CNAME": "ja3mamtora.github.io/Resume-Portfolio/"
```
Comment (DEV-DIBSTER) [inline domains/ja3mamtora.json:7]: CNAME cannot contain /.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Pull request closed without merging**